### DISCUSSION TITLE
Is there a way to set windynamicdesktop only for lockscreen and not for desktop, on Windows?

### DISCUSSION THREAD

**Discussion by pacman2108** [t1m0thyj/WinDynamicDesktop#460] [Q&A]
Is there a way to set windynamicdesktop only for lockscreen and not for desktop, on Windows?

**Comment by AnAceCase** [comment]
> Is there a way to set windynamicdesktop only for lockscreen and not for desktop, on Windows?

At the Moment, No this is as WinDD works by modifying the Desktop Wallpaper.

Put in a feature request in the issues tab and it might be considered to be added however
